week0/homework/PersonES5.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function Person(name, age) {
+	this.name = name;
+	this.age = age;
+	this.addPhone = function(phone) {
+		this.phone = phone;
+	};
+
+	this.call = function() {
+		if (this.phone) {
+			console.log('calling ' + this.name + ' at ' + this.phone + '...');
+		} else {
+			console.log(this.name + ' has no phone number saved.');
+		}
+	};
+
+	this.birthday = function() {
+		console.log(
```

Comment:
@jawkha, here's the only thing i was expecting a bit differently, was to increment the age, and not just display the incremented age, but of course it wasn't specifically mentioned in the instruction.